test/e2e/framework/create.go

```diff
@@ -34,6 +34,8 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/test/e2e/framework/testfiles"
+
+	imageutils "k8s.io/kubernetes/test/utils/image"
```

Comment:
Why the package rename? The author of that package called it just 'image'. Unless there's a good reason (like a conflict with some other package) I wouldn't change that.

Reply:
I just oriented at the other packages consuming utils/image, all of these use this package rename (a `grep -R -i -I --include "*.go" "\/test\/utils\/image" test` shows this. I thought this was some kind of convention. I'm fine with changing this.

Reply:
No, if everyone does that, then keep it.